packages/ts-moose-lib/package.json

```diff
@@ -19,6 +19,7 @@
     "test": "mocha -r ts-node/register 'src/**/*.test.ts'"
   },
   "dependencies": {
+    "@aws-sdk/client-s3": "^3.832.0",
```

Comment:
I don't think we should have the S3 source as part of `moose-lib` that should be a new package like:

`514labs/moose-s3` otherwise, for each connector and integration we will have a new dependency here and it will be bloated for people who don't use it.

Reply:
Good call. I've moved it to its own package & tested with npm link locally. Updated the build scripts as well, I think it should just publish when it's merged to main.